grafana-plugin/src/pages/settings/tabs/Cloud/CloudPage.tsx

```diff
@@ -15,7 +15,7 @@ import { WithStoreProps } from 'state/types';
 import { useStore } from 'state/useStore';
 import { withMobXProviderContext } from 'state/withStore';
 import { openErrorNotification } from 'utils';
-import { determineRequiredAuthString, UserActions } from 'utils/authorization';
+import { UserActions, determineRequiredAuthString } from 'utils/authorization';
```

Comment:
just curious...do you intentionally change import order? (There are several cases in your PR)

Reply:
VS Code has now builtin capability to automatically sort & remove imports on save and I have it enabled. 
https://dev.to/shanesc/how-to-sort-and-cleanup-imports-on-save-in-vs-code-52p1 

I guess it might be worth to run it across the whole codebase in one PR at some point and then maybe agree that we all use it. Then we will have all the imports automatically sorted on save and such diffs will not be present in PRs

Reply:
not everybody in our team uses VS Code,  we use  eslint `import` plugin for that (see details here https://github.com/grafana/oncall/blob/dev/grafana-plugin/.eslintrc.js#L13), I propose to keep using eslint for such things

Reply:
I agree if not everyone is using VS Code, we should definitely go for solution that will be consistent for everyone, no matter what IDE we're using.

But maybe inn this case maybe worth to consider https://www.npmjs.com/package/prettier-plugin-organize-imports ? 

I mean typically you format code with Prettier on save. You don't run `eslint --fix` on save, right? And I guess it's better to have those imports organized/sorted/deleted when you code rather than on some precommit hook or on CI

Reply:
I use precommit hook for formatting, it' ok for me...yes, we can start using https://www.npmjs.com/package/prettier-plugin-organize-imports but as for me it is optional, if you think this will improve our DX then feel free to add a PR for that =)